docs/userguide/containers/dockerrepos.md

```diff
@@ -42,6 +42,7 @@ You can now commit and push your own images up to your repos on Docker Hub.
 > **Note:**
 > Your authentication credentials will be stored in the `~/.docker/config.json`
 > authentication file in your home directory.
+> For Windows users, this Docker file is stored in `$HOME/.docker/config.json`
```

Comment:
Using `$HOME/.docker/config.json` on Windows assumes that the end user is using bash, not native cmd or PowerShell. It's `%HOME%\.docker\config.json` for cmd users, and `$env:Home\.docker\config.json` for PowerShell users.